pages/code_journal.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
Don't be like me and only do 2 git commits for a project :-). It would be great to see a few more commits for a project this size, and as you begin doing larger projects and working in Rails, you'll definitely want to commit often with detailed commit messages.